frontend/packages/vkt/public/i18n/fi-FI/common.json

```diff
@@ -22,11 +22,14 @@
       "edit": "Muokkaa",
       "errors": {
         "api": {
-          "createReservationCongestion": "Tutkintotilaisuus on ruuhkautunut",
-          "createReservationRegistrationClosed": "Tutkintotilaisuuteen ilmoittautuminen on sulkeutunut",
           "enrollmentMoveExamEventLanguageMismatch": "Ilmoittautumisen siirto toiseen tilaisuuteen epäonnistui, koska tilaisuuden kieli ei ole sama.",
           "examEventDuplicate": "Tutkintotilaisuus on jo olemassa jolla sama kieli, taso ja päivä",
           "generic": "Toiminto epäonnistui, yritä myöhemmin uudelleen",
+          "initialiseEnrollmentHasCongestion": "Tutkintotilaisuus on ruuhkautunut",
+          "initialiseEnrollmentHasQueue": "Tutkintotilaisuuteen ei ole enää paikkoja",
+          "initialiseEnrollmentIsFull": "Tutkintotilaisuuteen ei ole enää paikkoja",
```

Comment:
Jäi miettiin, että mikä on konseptuaalinen ero sillä, että tutkintotilaisuus on täynnä vs tutkintotilaisuuten oon jonoa? Missä tuota eroavaisuutta käytettiin jotenkin hyväksi? Naivi kysymys, että riittäisikö pelkkä "tutkintotilaisuus on täynnä" handlaamaan nää molemmat?

Reply:
Jos tutkintotilaisuus on mennyt täyteen ja joku ilmoittautuu jonoon ja sitten perutaan x kpl paikkoja, tilaisuus ei ole täynnä mutta siihen on jonoa. Tällöin uusi ilmoittautuja ei voi päästä jonossa olevan ohi tutkintoon, vaan päätyy myös jonoon (ellei jonossa olevaa nosteta tilaisuuteen ensin).

Käyttäjän näkökulmasta tuntui loogiselta, että molemmissa tuo toastin viesti nyt sattuu olemaan että tilaisuuteen ei enää paikkoja. Sen puolesta bäkkärikin vois ehkä palauttaa saman apiexceptionin molemmille keisseille. Mut ehkä mä mergeän tän nyt näin?

Reply:
PublicExamEventServicessä tällainen rivi:
`final long openings = examEventIdsHavingQueue.contains(e.id()) ? 0L : e.maxParticipants() - e.participants();`

Tuo tosiaan määrittää sen, että frontend näyttää molemmissa näistä tapauksista että tilaisuuteen ei yhtään paikkaa.

Reply:
Itse asiassa tässä ehdossa kun just määritellään, että openings perustuu myös jonopaikkojen olemassaoloon, niin jos mä kuitenkin muokkaan tuota PublicEnrollmentServiceä vielä ja siten että sieltä tulee sama APIException näissä molemmissa tapauksissa.